fjord/feedback/static/js/generic_feedback_dev.js

```diff
@@ -139,6 +139,9 @@
                 $('#browser-ask').show();
                 remoteTroubleshooting.getData(function (data) {
                     browserData = data;
+                    if ($('#browser-ok').is(':checked')) {
+                        $('#browser-data').val(JSON.stringify(browserData));
+                    }
```

Comment:
Had to add this because otherwise with the current code the `#browser-data` only gets populated if you touch the checkbox. We only want the data to be populated if the box is checked. Probably a better way to do this, but it'd involve rewriting more stuff.